src/components/header/Header.js

```diff
@@ -1,29 +1,48 @@
 import React from "react";
 import { Col, Input, Row } from "antd";
 import Logo from "./Logo";
+import lonceng from "../../assets/img/lonceng.png";
```

Comment:
Please rename `lonceng` to `notificationIcon`. Let's use English as much as possible.

Reply:
ok